site/en/integrations/integrate_with_voyageai.md

```diff
@@ -0,0 +1,152 @@
+---
+id: integrate_with_openai.md
+summary: This page discusses vector database integration with OpenAI's embedding API.
```

Comment:
summary: This page discusses vector database integration with VoyageAI's embedding API.

Reply:
Interestingly, PyCharm keeps these 2 lines hidden by default. :-)
Anyway, i just corrected these 2 lines